src/commcare_cloud/ansible/roles/ecryptfs/tasks/tmp.yml

```diff
@@ -1,4 +1,31 @@
 ---
+- name: Create {{encrypted_tmp}} to use for as tmp directory.
+  become: yes
+  file:
+    path: "{{ encrypted_tmp }}"
+    mode: "u=rwx,g=rwx,o=rx"
+    state: directory
+    owner: cchq
+    group: cchq
+
+  tags:
+    - mount-ecryptfs
+    - after-reboot
+    - tmp-encrypt
+
+- name: Create purging cron jobs
+  become: yes
+  cron:
+    name: "Purge files in {{ item }}"
+    special_time: daily
+    job: "/usr/sbin/tmpreaper 2d {{ item }}"
+    user: root
+    cron_file: purge_tmp_files
+  with_items:
+    - "{{ encrypted_tmp }}"
```

Comment:
`encrypted_tmp` is always set to `/opt/tmp` regardless of the `root_encryption_mode` setting [https://github.com/dimagi/commcare-cloud/blob/b01255bc73fb67bdd52284612b2feb8954fd6556/src/commcare_cloud/ansible/group_vars/all.yml#L13](here). We stopped creating that directory entirely, and this change makes us create it regardless of the `root_encryption_mode` setting. When `root_encryption_mode == 'ecryptfs'`, we do the ecryptfs below; otherwise (`root_encryption_mode == 'aws'`) we still want to create the directory, but it's already on an encrypted disk so no ecryptfs'ing is necessary.